src/app/toolbar/toolbar.component.scss

```diff
@@ -17,20 +17,26 @@
 .toolbar img {
   margin: 0 spacing.$space-normal;
   width: 10%;
-  padding: spacing.$space-normal;
 }
 
-.toolbar #twitter-logo {
-  height: 3rem;
-  margin: 0 8px;
+.toolbar a {
+  text-decoration: none;
+  color: white;
+  padding: spacing.$space-normal;
 }
 
-.toolbar #youtube-logo {
-  height: 3rem;
-  margin: 0 spacing.$space-normal;
-}
+// leave this comments for learning purposes
```

Comment:
I don't think we learn something from these classes. You can always check your git history to find this code in the future. It is a far superior approach to leaving commented code. Think of git as a historian. Once you commit something, it will remember it. So, even if you delete something, you can go back to git and find out deleted code.

I'm open to discussing this, I think it's an opportunity to learn something new.